### PR TITLE
Dummy customer detail views

### PR DESCRIPTION
**Pull request by juhakujala**

### CODE REVIEW

**Review by frwickst — CHANGES_REQUESTED** [review]
This was split up quite nicely and since there were so many different things in here I can't really say that I have much to complain about here.
The only thing that could help is to create the utility function that I mentioned in the comments. Other then that this is OK by me.

File: src/components/installments/InstallmentsTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      </Dialog.ActionButtons>
+    </Dialog>
+  );
+
+  return (
+    <tr>
+      <td>{t(`ENUMS.${installment.type}`)}</td>
+      <td style={{ textAlign: 'right' }}>
+        {installment.amount
+          ? Intl.NumberFormat('fi-FI', { style: 'currency', currency: 'EUR' }).format(installment.amount)
```
[the remark this answers is not in the document]
Comment (juhakujala) [reply src/components/installments/InstallmentsTableRow.tsx:89]: Yep, I can also add one for these values 👍

[the remark this answers is not in the document]
Comment (juhakujala) [reply src/components/installments/InstallmentsTableRow.tsx]: Updated [here](https://github.com/City-of-Helsinki/att-sales-ui/pull/19/commits/0b19c3fc2a2111886856f5b0055c1f65ed59e198)

**Review by frwickst — APPROVED** [review]
Nice that you made it a bit more generic, good change 👍  🚢

**Pull request merged**